Contest_API.md

```diff
@@ -1797,19 +1797,19 @@ The following endpoints are associated with clarification messages:
 
 Properties of clarification message objects:
 
-| Name           | Type    | Description
-| :------------- | :------ | :----------
-| id             | ID      | Identifier of the clarification.
-| from\_team\_id | ID ?    | Identifier of [team](#teams) sending this clarification request, `null` iff a clarification sent by jury.
-| to\_team\_id   | ID ?    | Identifier of the [team](#teams) receiving this reply, `null` iff a reply to all teams or a request sent by a team.
-| reply\_to\_id  | ID ?    | Identifier of clarification this is in response to, otherwise `null`.
-| problem\_id    | ID ?    | Identifier of associated [problem](#problems), `null` iff not associated to a problem.
-| text           | string  | Question or reply text.
-| time           | TIME    | Time of the question/reply.
-| contest\_time  | RELTIME | Contest time of the question/reply.
+| Name           | Type             | Description
+| :------------- | :--------------- | :----------
+| id             | ID               | Identifier of the clarification.
+| from\_team\_id | ID ?             | Identifier of the [team](#teams) sending this clarification request, `null` iff a clarification is sent by the jury.
+| to\_team\_ids  | array of ID ?    | Identifiers of the [team(s)](#teams) receiving this reply, `null` iff a reply to all teams or a request sent by a team.
```

Comment:
Should we actually remove the `?` and make the empty array be the replacement? If not, we should specify that if you set the value to non-null, it should (must?) not be the empty array?

Reply:
Just as a side note while thinking about this: I think we've always accepted not returning a property when it's `null`. I can't find it in the current draft anymore, did we remove this?

Reply:
https://ccs-specs.icpc.io/2023-06/contest_api#extensibility

Reply:
> Should we actually remove the `?` and make the empty array be the replacement? If not, we should specify that if you set the value to non-null, it should (must?) not be the empty array?

I really don't like the empty list having the meaning "all".

Reply:
Discussed at meeting. Will not do.